Plugins/BCClient/Source/BCClientPlugin/Private/BrainCloudEvent.cpp

```diff
@@ -44,6 +44,35 @@ void BrainCloudEvent::deleteIncomingEvent(const FString &evId, IServerCallback *
 	_client->sendRequest(sc);
 }
 
+void BrainCloudEvent::deleteIncomingEvents(const TArray<FString>& eventIds, IServerCallback* callback)
+{
+	TSharedRef<FJsonObject> message = MakeShareable(new FJsonObject());
+	message->SetArrayField(OperationParam::EventServiceEvIds.getValue(),JsonUtil::arrayToJsonArray(eventIds));
+
+	ServerCall *sc = new ServerCall(ServiceName::Event, ServiceOperation::DeleteIncomingEvents, message, callback);
```

Comment:
I cannot see the test because it's a binary blueprint file, but can you verify from the log that this callback with the deleted event you sent? I want you to comfirm the way we construct the JSON here is proper and all.

Reply:
I can verify that it does delete events that I've sent in. I can provide the log if needed.